src/components/manage/Blocks/Search/components/SortOn.jsx

```diff
@@ -73,7 +74,10 @@ const SortOn = (props) => {
           placeholder={intl.formatMessage(messages.sortOn)}
           styles={sortOnSelectStyles}
           theme={selectTheme}
-          label={intl.formatMessage('Sort by selected option')}
+          label={intl.formatMessage({
+            id: 'sort_by_selected_option',
```

Comment:
You should move this to messages var for better readability.

Reply:
do you have a suggestion for the variable name?

Reply:
See three lines previous for an example to follow. Again, you must follow the excellent documentation for to internationalize messages at https://6.docs.plone.org/volto/recipes/i18n.html.